one more panic

```
~/.gobrew/bin
 21:03:58  ❯ ./gobrew version
[INFO] gobrew version is dev

~/.gobrew/bin
 21:04:12  ❯ gobrew ls
panic: runtime error: index out of range [2] with length 0

goroutine 1 [running]:
github.com/kevincobain2000/gobrew.(*GoBrew).ListVersions(0xc000137f00)
	/home/runner/work/gobrew/gobrew/gobrew.go:149 +0x8f2
main.main()
	/home/runner/work/gobrew/gobrew/cmd/gobrew/main.go:70 +0x18a
```

--- a/gobrew.go
+++ b/gobrew.go
@@ -146,10 +146,12 @@ func (gb *GoBrew) ListVersions() {
 		version := versionSemantic.String()
 		// 1.8.0 -> 1.8, if version < 1.21.0
 		reMajorVersion := regexp.MustCompile("([0-9]+).([0-9]+).0")
-		vv, _ := strconv.Atoi(reMajorVersion.FindStringSubmatch(version)[2])
-		if vv < 21 {
-			if reMajorVersion.MatchString(version) {
-				version = strings.Split(version, ".")[0] + "." + strings.Split(version, ".")[1]
+		if len(reMajorVersion.FindStringSubmatch(version)) > 1 {
+			vv, _ := strconv.Atoi(reMajorVersion.FindStringSubmatch(version)[2])
+			if vv < 21 {
+				if reMajorVersion.MatchString(version) {
+					version = strings.Split(version, ".")[0] + "." + strings.Split(version, ".")[1]
+				}
 			}
 		}
 		if version == cv {

